### PR TITLE
LPS 146790 Can create a collection with no asset types configured

### PR DESCRIPTION
**Pull request by Stephy6**
Motivation
=======
It shouldn't be possible to create a manual collection of asset types if there are no asset types selected
[LPS-146790](https://issues.liferay.com/browse/LPS-146790)

Proposed Solution
========
When creating a new collection of multiple asset types, there are all assets type selected by default and the save button for that collection is enabled. When there are no asset types selected the save button should be disabled but it isn't. The proposed solution consists in disabling the save button when there are no asset types selected for that collection.

Steps to Verify:
----------------
1. Go to Site Builder > Collections
2. Create a manual collection
3. Item Type -> Multiple item types -> Select types...
4. Move all the asset types from the 'Selected' box to the 'Available' box
5. Check if the save button is disabled so it's not possible to create the collection

### CODE REVIEW

**Comment by ealonso** [comment]
ci:test:sf

**Comment by ealonso** [comment]
Hey @Stephy6,

Could you please fix the sf errors? You should execute format source before sending a pr.

Regards,
Eudaldo.

**Comment by Stephy6** [comment]
Hi @ealonso ,

SF executed.

Regards,
Stefan.

**Comment by victorg1991** [comment]
LGTM 🎉  Working great!! 👏

**Comment by victorg1991** [comment]
ci:forward

File: modules/apps/asset/asset-list-web/src/main/resources/META-INF/resources/js/Source.js [hunk 1]
```diff
@@ -186,7 +186,6 @@ export default function ({classTypes, namespace}) {
 		if (classTypeSelected) {
 			selectedSubtype = subtypeSelector[classTypeSelected.className];
 		}
-
```
Comment (ealonso) [inline modules/apps/asset/asset-list-web/src/main/resources/META-INF/resources/js/Source.js:189]: @Stephy6 @victorg1991 I think this deletion is no needed

**Pull request closed without merging**